sample_code_submission/model.py

```diff
@@ -226,20 +233,21 @@ def balance_set(self):
         for i in range(len(class_weights_train)):  # loop on B then S target
             # training dataset: equalize number of background and signal
             weights_train[train_labels == i] *= (
-                max(class_weights_train) / class_weights_train[i]
+                    max(class_weights_train) / class_weights_train[i]
             )
             # test dataset : increase test weight to compensate for sampling
 
         balanced_set["weights"] = weights_train
 
         return balanced_set
 
-    def predict(self, test_set):
+    def predict(self, test_set, stat_only: bool = None, syst_settings: dict[str, bool] = None):
```

Comment:
Are you sure this work without changing ingestion ? 
If not I would recommend using a different approach else it will be difficult to test it with codabench.org

Reply:
Yes. This works perfectly with the current ingestion program.